admin/app/controllers/authenticated/complementary-certifications/complementary-certification/attach-target-profile.js

```diff
@@ -9,16 +9,34 @@ export default class AttachTargetProfileController extends Controller {
 
   @tracked options = [];
   @tracked selectedTargetProfile;
+  @tracked targetProfileBadges = [];
+
+  get getTargetProfileBadgesErrorMessage() {
+
+    if(!this.targetProfileBadges?.length > 0){
```

Comment:
Etant donné qu'il est initialisé à tableau vide, pas besoin de checké si il est undefined

```suggestion
    if(!this.targetProfileBadges.length > 0){